src/interface/obsidian/src/chat_modal.ts

```diff
@@ -337,17 +360,12 @@ export class KhojChatModal extends Modal {
                     referenceExpandButton.innerHTML = expandButtonText;
                     references.appendChild(referenceSection);
                 } else {
-                    if (responseText.startsWith("{") && responseText.endsWith("}")) {
-                    } else {
-                        // If the chunk is not a JSON object, just display it as is
-                        continue;
-                    }
-
-                    this.renderIncrementalMessage(responseElement, responseText);
+                    await this.renderIncrementalMessage(responseElement, responseText);
```

Comment:
This was the fix for the chat streaming issue, right?

Reply:
Yes